src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -7,7 +7,7 @@ Feature: Artifact system test
     And some subscriptions are set up
     And a jenkins job '"ArtC2Job"' from '"src/systemtest/resources/JenkinsShellScripts/ArtC2Script.txt"' is created
```

Comment:
Instead of having n number of and, could this be done with a "Scenario Outline" where and is repeated for n number of defined jobs and from files?
https://www.baeldung.com/cucumber-scenario-outline

Reply:
But they are all required before the "then" statement so i don't know if we can put them in different scenarios like that. I don't think the example you sent has the same setting. Our scenarios would be Test artifact flow, Test testexecution flow etc but that wouldn't help much.